core/src/test/java/org/apache/iceberg/TestScanPlanningAndReporting.java

```diff
@@ -215,19 +215,24 @@ public void scanningWithSkippedDataFiles() throws IOException {
     Table table =
         TestTables.create(
             tableDir, tableName, SCHEMA, SPEC, SortOrder.unsorted(), formatVersion, reporter);
-    table.newAppend().appendFile(FILE_A).appendFile(FILE_D).commit();
-    table.newAppend().appendFile(FILE_B).appendFile(FILE_C).commit();
+    table.newAppend().appendFile(FILE_A).appendFile(FILE_B).appendFile(FILE_D).commit();
```

Comment:
I need to add `FILE_B` here so that we have a manifest with one skipped data file as the original manifest will be rewritten with `FILE_A` as deleted.